testapp/tests/test_behaviour_for_postgres.py

```diff
@@ -102,3 +102,14 @@ def test_executing_multiple_querysets_which_are_always_empty_is_handled(self):
             self.assertEqual(len(results), 2)
             self.assertEqual(results[0], [])
             self.assertEqual(results[1], [])
+
+    def test_fetch_count(self):
```

Comment:
add usage instructions for .count() in readme (mention it requires different approach since it is not lazy by default)

Reply:
Check readme